app/models/attestation_template.rb

```diff
@@ -128,17 +128,22 @@ def replace_type_de_champ_tags(text, types_de_champ, dossier_champs)
         .select { |dossier_champ| dossier_champ.libelle == tag[:libelle] }
         .first
 
-      acc.gsub("--#{tag[:libelle]}--", champ.to_s)
+      replace_tag(acc, tag, champ)
     end
   end
 
   def replace_tags_with_values_from_data(text, tags, data)
     if data.present?
       tags.inject(text) do |acc, tag|
-        acc.gsub("--#{tag[:libelle]}--", data.send(tag[:target].to_sym).to_s)
+        replace_tag(acc, tag, data.send(tag[:target].to_sym))
       end
     else
       text
     end
   end
+
+  def replace_tag(text, tag, value)
+    libelle = Regexp.quote(tag[:libelle]).gsub(/\\?[[:blank:]]/, "[[:blank:]]")
```

Comment:
Rajouter un commentaire pour expliquer ce qu'on fait, du genre "We replace any whitespace character by a generic whitespace regex matcher" (ou un truc plus précis / correct :) )